data/json/recipes/recipe_food.json

```diff
@@ -4251,6 +4259,235 @@
   ]
 },{
   "type" : "recipe",
+  "result": "syrup",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_DRINKS",
+  "skill_used": "cooking",
+  "difficulty": 4,
+  "time": 80000,
+  "autolearn": true,
+  "batch_time_factors": [ 60, 3 ],
+  "qualities":[
+    {"id":"BOIL","level":2}
+  ], "tools": [
+    [
+      [ "rag", -1 ]
+    ],
+    [
+      [ "hotplate", 12 ],
+      [ "toolset", 12 ],
+      [ "fire", -1 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "maple_sap", 35 ]
+    ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "maple_lollipop",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_SNACK",
+  "skill_used": "cooking",
+  "difficulty": 5,
+  "time": 30000,
+  "autolearn": false,
+  "book_learn": [[ "cookbook_maple", 5 ]],
+  "qualities":[
+    {"id":"BOIL","level":2}
+  ], "tools": [
+    [
+      [ "thermometer", -1 ]
+    ],
+    [
+      [ "hotplate", 8 ],
+      [ "toolset", 8 ],
+      [ "fire", -1 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "syrup", 2 ]
+    ],
+    [
+      [ "splinter", 1 ],
+      [ "skewer_bone", 1 ],
+      [ "skewer", 1 ]
+    ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "maple_candy",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_SNACK",
+  "skill_used": "cooking",
+  "difficulty": 5,
+  "time": 30000,
+  "autolearn": false,
+  "book_learn": [[ "cookbook_maple", 5 ]],
+  "qualities":[
+    {"id":"BOIL","level":2}
+  ], "tools": [
+    [
+      [ "thermometer", -1 ]
+    ],
+    [
+      [ "hotplate", 8 ],
+      [ "toolset", 8 ],
+      [ "fire", -1 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "syrup", 2 ]
+    ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "maple_taffy",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_SNACK",
+  "skill_used": "cooking",
+  "difficulty": 4,
+  "time": 30000,
+  "autolearn": true,
+  "batch_time_factors": [ 60, 3 ],
+  "qualities":[
+    {"id":"BOIL","level":2}
+  ], "tools": [
+    [
+      [ "hotplate", 8 ],
+      [ "toolset", 8 ],
+      [ "fire", -1 ]
+    ]
+  ],
+  "components": [
+    [
+      [ "syrup", 10 ]
+    ]
+  ]
+},{
+  "type" : "recipe",
+  "result": "maple_tenderloin",
+  "category": "CC_FOOD",
+  "subcategory": "CSC_FOOD_MEAT",
+  "skill_used": "cooking",
+  "difficulty": 5,
+  "time": 45000,
+  "reversible": false,
+  "autolearn": false,
+  "book_learn": [[ "cookbook_maple", 5 ]],
+  "qualities":[
+    {"id":"BUTCHER","level":20},
```

Comment:
Butcher quality in crafting? It's the first such usage. Not technically wrong, but unusual.